homeassistant/components/meteoclimatic/weather.py

```diff
@@ -0,0 +1,151 @@
+"""Support for Meteoclimatic weather service."""
```

Comment:
There is a sensor and weather platform in this PR. Please limit PRs to adding a single platform.

So either limit it to the sensor or to weather for this PR. Additional platforms can be added in a follow-up PR.